### DISCUSSION TITLE
Release 1.16.0

### DISCUSSION THREAD

**Discussion by BrightDV** [BrightDV/BlackHole#1] [Announcements]
**This is my first release of BlackHole so expect some bugs!**

### Note that there are some regressions in the app because some packages' forks with bug fixes used by the app have been removed by the original owner.

# Changelog
- Fix the search
- Support listening to podcasts!
- Improve Top Searches layout
- Add a recommendation row to the album view
- Localized search results categories
- New player UI: lyrics at the bottom, queue in a new screen (button at top right)
- Some other UI tweaks

<hr /><em>This discussion was created from the release <a href='https://github.com/BrightDV/BlackHole/releases/tag/v1.16.0'>Release 1.16.0</a>.</em>

**Comment by ghost** [comment]
Hey @BrightDV, thanks for doing this. I use this app regularly and was planning to fix the search issue myself but found this instead. I'd love to help out. Let me know if you need help with anything.

**Reply by BrightDV** [reply]
Thanks for wanting to help!
I really don't have that much time, but you can still open PRs; I will try to review and merge them when I have time. However, don't expect more than one release per month.

If you want to fix bugs, check #3 (or #4, but I think that this one only needs a bump of `youtube_explode` to the latest version).
